code/datums/diseases/mob_procs.dm

```diff
@@ -142,8 +142,8 @@
 			if(isobj(wear_suit))
 				Cl = wear_suit
 				. = prob((Cl.permeability_coefficient*100) - 1)
-			if(. && isobj(WEAR_BODY))
```

Comment:
why have you changed this? this has nothing to do with pronouns, split this off into a separate PR please